docs/user/alerting/stack-alerts/es-query.asciidoc

```diff
@@ -45,22 +45,24 @@ When the ES query alert condition is met, the following variables are available
 `context.value`:: The value of the alert that met the condition.
 `context.conditions`:: A description of the condition. Example: `count greater than 4`.
 `context.hits`:: The most recent ES documents that matched the query. Using the https://mustache.github.io/[Mustache] template array syntax, you can iterate over these hits to get values from the ES documents into your actions.
-
++
 [role="screenshot"]
 image::images/alert-types-es-query-example-action-variable.png[Iterate over hits using Mustache template syntax]
 
+<<defining-alerts-actions-details, Add an action>> to the alert to use these variables.
+
 [float]
-==== Testing your query
+==== Test your query
```

Comment:
If "When your query is valid" and "When your query is invalid" are not part of the UI, then I would remove these titles.  Using the same format for this section as for the variable section is confusing.